### PR TITLE
fix(description) : changed description of the hints in step 40 of the skyline project

### LINKED ISSUE #47616: Lack of a descriptive enough hint in step 40 of building skyline project
In the hint below it ask to add variable, but it's similar to how we ask colors to be added, which cased confusion to a camper

https://github.com/freeCodeCamp/freeCodeCamp/blob/2e4f1e31be28cd9d8287dd0fdf110aab4af11dde/curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98f0.md#L35

I propose, applying the same approach in ieahleen pull request to skyline variables name, #47609. Although it feels like giving answers outright, variables are case-sensitive and require specific syntax, which I think we should give their steps some lay way in the hints. 

original issue: https://freecodecamp.crowdin.com/translate/690087755819f2f67b26dbee3be0eb53/32714/en-esem?filter=basic&value=0#q=1153282

### PR DESCRIPTION
**Pull request by yashkoli836**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #47616

### CODE REVIEW

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
Thank you for your PR. Please follow the checklist you have checked, and give the PR a descriptive title.

You may have missed it, we have [a whole section on how to create a good PR title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title)

**Comment by yashkoli836** [comment]
> Thank you for your PR. Please follow the checklist you have checked, and give the PR a descriptive title.
> 
> You may have missed it, we have [a whole section on how to create a good PR title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title)

Hey @ieahleen, I think now the PR title is correct pls let me know if there is another need of correction.

**Review by majestic-owl448 — APPROVED** [review]
Thank you for your PR, now LGTM 👍

File: curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98f0.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ You should give the `background` a `linear-gradient`.
 assert.include(new __helpers.CSSHelp(document).getStyle('.bb1a')?.background, 'linear-gradient');
 ```
 
-You should give the `background` a `linear-gradient` starting from `--building-color1`.
+You should give the `background` a `linear-gradient` starting from `--building-color1` variable.
```
Comment (jeremylt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98f0.md:35]: I do not see this as needed

**Review by jeremylt — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98f0.md (hunk 1 above)
Comment (ieahleen) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98f0.md:35]: @jeremylt It was being discussed in the issue as being confusing. The other alternative is 
```suggestion
You should give the `background` a `linear-gradient` starting from `var(--building-color1)`.
```

Comment (jeremylt) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98f0.md:35]: The current seems grammatically awkward.

```suggestion
You should give the `background` a `linear-gradient` that starts with the `--building-color1` variable.
```

Maybe?

Comment (Sboonny) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98f0.md:35]: The issue is still labeled discussion, so we can discuss what to do about it there.

**Comment by ieahleen** [comment]
I am closing this as the issue is stil labelled with https://github.com/freeCodeCamp/freeCodeCamp/labels/status%3A%20discussing, feel free to reopen once a consebnsus has been reached and the issue has been opened for cotnribution

**Pull request closed without merging**